.github/workflows/unit-test.yml

```diff
@@ -0,0 +1,51 @@
+name: UnitTests
+
+on:
+  pull_request:
+    branches: 
+     - "**"
+
+jobs:   
+  run-unit-tests:
+
+    runs-on: ubuntu-latest
+    container: compucorp/civicrm-buildkit:1.0
+    env:
+      CIVICRM_CORE: 5.24.6
+      SHOREDITCH_TAG: 0.1.0-alpha.37
+
+    services:
+      mysql:
+        image: mysql:5.7
+        env:
+          MYSQL_ROOT_PASSWORD: root
+        ports:
+        - 3306
+        options: --health-cmd="mysqladmin ping" --health-interval=10s --health-timeout=5s --health-retries=3
+
+    steps:
+
+      - uses: actions/checkout@v2
+
+      - name: Config mysql database
+        run: echo "SET GLOBAL sql_mode=(SELECT REPLACE(@@sql_mode,'ONLY_FULL_GROUP_BY',''));" | mysql -u root --password=root --host=mysql
+
+      - name: Config amp
+        run : amp config:set --mysql_dsn=mysql://root:root@mysql:3306
```

Comment:
is this step really necessary, given you already configure this [in the dockerfile](https://github.com/compucorp/civicrm-buildkit-docker/blob/3b77c7e70c26978f44475878e99e3b9106d4b709/Dockerfile#L75)?

Reply:
@davialexandre Yes, it is as what we have already configured it was for /buildkit so it won't be pickup with Github Action by configuring amp. The new services.yml will be created under /github/home/.amp/services.yml.  I have removed configured from docker image as we are not really using it.

Reply:
@erawat it would be great if we could have this pre-configured in the docker image and don't need to duplicate things in workflows. Is it really not possible to do that?

Reply:
@davialexandre  amp config is now removed from docker image and I am not sure if we create /github/home directory on the image will have any issue when running workflow or not. Perhaps we can leave it for the next / improvement?

Reply:
We chatted a bit more about this earlier today and agreed to keep this as is